src/js/server/hcapi/stats.js

```diff
@@ -31,24 +31,31 @@ statsRouter.get('/', (req, res, next) => {
     return parseInt(counts[0].count);
   });
 
+  const leaderboardQuery = `SELECT "admins"."name", COALESCE(reviews.count, 0) AS count, "admins"."id" FROM "admins" LEFT JOIN (SELECT "adminId", COUNT(DISTINCT "hackerApplicationId") FROM "application-reviews" GROUP BY "adminId") AS reviews ON "admins"."id" = "adminId" ORDER BY count DESC, "admins"."name" ASC`;
```

Comment:
Perhaps we could break out the query into a file and load it in with fs for readability?